admin_guide/quota.adoc

```diff
@@ -85,6 +81,28 @@ exceed this value.
 |===
 
 
+.Storage Resources Managed by Quota
+[cols="3a,8a",options="header"]
+|===
+
+|Resource Name |Description
+
+|`*requests.storage*`
+|The sum of storage requests across all persistent volume claims in any state cannot
+exceed this value. `*storage*` and `*requests.storage*` are the same value and can
```

Comment:
the only supported value is `requests.storage`  - i am not sure why we stated `storage` as interchangeable.  it never has been.